index.html

```diff
@@ -138,16 +138,18 @@
     */
     
     // Working group info
-    wg:           "Accessible Rich Internet Applications Working Group",
-    wgURI:        "https://www.w3.org/WAI/ARIA/",
-    wgPublicList: "public-aria",
+	group: "aria",
+
+    // wg:           "Accessible Rich Internet Applications Working Group",
+    // wgURI:        "https://www.w3.org/WAI/ARIA/",
+    // wgPublicList: "public-aria",
```

Comment:
Shouldn't we just delete these rather than comment them out?

Reply:
+1